docs/getting-started/style-guide.html

```diff
@@ -156,11 +156,18 @@ <h2 class="heading-2"><span class="blue-text">Step 5:</span> Defining a Book mod
           <pre><code>open Sources/Application/Models/Book.swift</code></pre>
           <p>Inside this file we define our Book:</p>
```

Comment:
Not on this line (sorry), please can you encase `Book.swift` in backticks throughout the style guide.